scripts/apps/authoring/authoring/services/AuthoringService.ts

```diff
@@ -497,7 +497,8 @@ export function AuthoringService($q, $location, api, lock, autosave, confirm, pr
             helpers.filterDefaultValues(diff, origItem);
 
             if (_.size(diff) > 0) {
-                return api.save('archive', origItem, diff).then((__item) => {
+                return api.save('archive', origItem, diff, {},
+                    {publish_from_personal: appConfig?.features?.publishFromPersonal}).then((__item) => {
```

Comment:
wouldn't this actually work even if it's set to `false` ?

Reply:
I think It would work.